src/Widgets/PasswordEditor.vala

```diff
@@ -107,20 +103,26 @@ namespace SwitchboardPlugUserAccounts.Widgets {
                 error_new_revealer.set_transition_duration (200);
                 error_new_revealer.set_reveal_child (false);
                 error_new_revealer.add (error_new_label);
+
+                attach (current_pw_entry, 0, 0, 1, 1);
+                attach (error_revealer, 0, 1, 1, 1);
                 attach (error_new_revealer, 0, 3, 1, 1);
 
-            } else if (get_permission ().allowed)
+            } else if (get_permission ().allowed) {
```

Comment:
I know this wasn't meant to be changed, but since we are here, can this `get_permission ()` be assigned to some temporary `permission` variable and use that instead? We are currently calling `get_permission ()` multiple times, and so multiple checks have to be done for permission to get it.